includes/customer-functions.php

```diff
@@ -217,6 +217,11 @@ function edd_get_customers( $args = array() ) {
 		'number' => 30
 	) );
 
+	if ( -1 == $args['number'] ) {
```

Comment:
```suggestion
	if ( -1 == $r['number'] ) {
```

This needs to be `$r` because `$args` is only looking at the value passed in by the user, which may not contain `number` at all. `$r` is after we've merged the supplied arguments with our defaults.

Fixing this will fix the unit tests complaining about :

```
Undefined index: number
/home/travis/build/easydigitaldownloads/easy-digital-downloads/includes/customer-functions.php:220
```

Reply:
Thanks, I just caught that--fixed in latest.